src/Request/RequestHandler.php

```diff
@@ -428,6 +441,9 @@ protected function sortParameters(array $parameters)
 
     /**
      * Escapes all special symbols for query
+     * All parameter names and values are escaped using the percent-encoding (%xx) mechanism.
+     * Characters not in the unreserved character set ([RFC3986] section 2.3) MUST be encoded.
+     * Characters in the unreserved character set MUST NOT be encoded.
```

Comment:
We don't need `<` or `>` or `{` or `}` characters, we need to reject all characters that are not needed :)